homeassistant/components/starlink/strings.json

```diff
@@ -0,0 +1,17 @@
+{
+  "config": {
+    "abort": {
+      "already_configured": "Starlink terminal is already configured"
```

Comment:
Please use the standard `key::common` messages here

Reply:
By using the `key::common` strings we don't place additional work burden on translators